stable/admin/templates/statefulset.yaml

```diff
@@ -27,8 +27,15 @@ spec:
   template:
     metadata:
       name: {{ template "admin.fullname" . }}
-      {{- if .Values.admin.podAnnotations }}
       annotations:
+      {{- if .Values.admin.configFiles }}
+        {{- with .Values.admin.configFiles }}
+        checksum/config: {{ toYaml . | sha256sum }}
```

Comment:
What if we have more than 1 config file? Shouldn't we include the config file "key" in the annotation key?